.github/workflows/rotate-emoji.yml

```diff
@@ -0,0 +1,53 @@
+name: Rotate emojiData
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: rotate-emoji-group
+
+jobs:
+  sync-emojis-from-github:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js v20
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
+      - name: Fetch emojiData
+        run: npm run build:emoji
+
+      - name: Check for changes
+        run: |
+          if git diff --quiet -- "src/core/render/emoji-data.js"; then
+            echo "No changes Found. Skipping PR creation."
+          else
+            echo "Changes detected in emoji-data.js. Creating PR."
```

Comment:
Why not just commit instead of using PR?

Reply:
As per the previous discuss, we'd better have a review on the changes instead of commit it directly.